.stylelintrc.json

```diff
@@ -4,5 +4,16 @@
   ],
   "plugins": [
     "stylelint-scss"
-  ]
+  ],
+  "rules": {
+    "at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "include", "extend"
+        ]
+      }
+    ],
+    "scss/at-rule-no-unknown": true
```

Comment:
are you sure you also need this line ?

Reply:
If you want a validation yes, previous lines are just to disabled default checks